src/components/Layouts/_index.scss

```diff
@@ -0,0 +1,68 @@
+////
+/// Modular layouts styles.
+/// @group modular-layouts
+/// @copyright IBM Security 2020
+////
+
+@import '@carbon/layout/scss/spacing';
+@import '@carbon/themes/scss/theme-maps';
+
+@import '../../globals/namespace/index';
+
+/// Generate a map of 'layout' spacing tokens from the provided Carbon theme map.
+/// @access private
+/// @param {Map} $theme-map - Map of theme variables.
+/// @returns {Map} - The filtered map of only spacing variables.
+@function security--get-spacing-tokens-map($theme-map) {
+  $-map: ();
+
+  @each $token, $value in $theme-map {
+    @if type-of($value) == 'number' {
+      @if str-index($token, 'layout-') {
```

Comment:
@SimonFinney I added an extra change here because I noticed that we were getting a number of class names generated that we probably would never realistically used, like for icon spacing for larger "container" spacing. If we want to expand the scope, we could update this check in the future. But we now, I think we should keep the scope small to just `layout-` tokens.